spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe CommentsHelper do
+  describe "#comment_path" do
+    let(:comment) { VCR.use_cassette('planningalerts') { create(:confirmed_comment) } }
+
+    it "returns the path for the application with an anchor for the comment" do
+      expect(helper.comment_path(comment)).to eq application_path(comment.application, anchor: "comment#{comment.id}")
```

Comment:
While you could certainly argue that this test is just testing the framework, you're not testing anything if you generate both sides of a test. It's like saying `expect(1 + 1).to eq (1 + 1)`, it needs to be `expect(1 + 1).to eq 2`.

It's up to you if you want to remove the test completely or fix this test.